numpy/lib/scimath.py

```diff
@@ -403,7 +403,7 @@ def log2(x):
 
     Returns
     -------
-    out : ndarray or scalar
+    out : array_like
```

Comment:
In my opinion, this change would actually make the docs less clear as this makes the return type less specific. `array_like` is typically used for *inputs* to indicate that objects which *can* be treated as arrays (or scalars) by numpy *will be* internal to the function. Numpy functions almost always return arrays or scalars but not other sequence types (like lists) so `ndarray or scalar` is actually a better description for the return type IMO.